src/include/sof/trace/trace.h

```diff
@@ -21,7 +21,7 @@
 #endif
 #include <sof/common.h>
 #include <rtos/sof.h>
-#include <sof/trace/preproc.h>
+#include <preproc/preproc.h>
```

Comment:
Looks like next steps could be to move trace.h to xtos header too since it will be all xtos specific APIs.

Reply:
trace.h header file both include zephyr and non-zephyr definitons, if want to move to xtos/zephyr directory, we need duplicate it like other header files under xtos/include or zephyr/include.